app/controllers/requests/request_controller.rb

```diff
@@ -38,16 +37,6 @@ def aeon?
       false
     end
 
-    def redirect_single_aeon_thesis_numismatics
```

Comment:
After we remove this, do we still see the behavior where numismatics and thesis holdings redirect you directly to Aeon?

Reply:
Yes they go through AEON

Reply:
@kevinreiss do you mean if a user goes directly to https://catalog.princeton.edu/requests/coin-876?aeon=false ? If so, it doesn't seem to be redirecting in production currently.

Reply:
whoops, scratch that.  https://catalog.princeton.edu/requests/coin-876?aeon=false&mfhd=numismatics does still redirect in prod, but not in our branch.  Is this redirect behavior needed?

Reply:
@christinach and I looked at the http://localhost:3000/requests/coin-876?aeon=false&mfhd=numismatics together, and we don't think it presents a problem.  The show page does not link to that URL, and a user probably couldn't have bookmarked it, since it's a redirect.